react-router/index.js

```diff
@@ -1,2 +1,29 @@
 export { default } from 'react-router';
-export * from 'react-router';
+export {
+  __esModule,
```

Comment:
and here

Reply:
I must skip the export script to expose `__esModule` key for these kind of dependencies that include it. I'll do.